src/annotator/util/shadow-root.js

```diff
@@ -43,5 +44,18 @@ export function createShadowRoot(container) {
     applyFocusVisible(shadowRoot);
   }
 
+  stopEventPropagation(shadowRoot);
   return shadowRoot;
 }
+
+/**
+ * Stop bubbling up of the 'click' and 'touchstart' events.
+ * @param {HTMLElement|ShadowRoot} element
+ */
+function stopEventPropagation(element) {
+  element.addEventListener('click', event => event.stopImmediatePropagation());
```

Comment:
Is the use of `stopImmediatePropagation` vs `stopPropagation` here important? From looking at the code in the `Guest` class it looks like it adds the event listener to `this.element` which is `document.body` (see annotator/index.js). I can see that it is needed with the way that the tests have been written.

Reply:
No, it is not important. Both will stop the propagation. `stopImmediatePropagation` in addition to stop the propagation will prevent other events attached to the same element to be called.
https://developer.mozilla.org/en-US/docs/Web/API/Event/stopImmediatePropagation

It is a subtle and in this context unimportant.

I will change `stopImmediatePropagation` for `stopPropagation` and fix the tests.